larq_compute_engine/mlir/BUILD

```diff
@@ -1,11 +1,20 @@
-load("@org_tensorflow//third_party/mlir:tblgen.bzl", "gentbl")
+load("@org_tensorflow//third_party/mlir:tblgen.bzl", "gentbl", "td_library")
 load("@org_tensorflow//tensorflow:tensorflow.bzl", "pybind_extension", "tf_cc_binary")
 
 package(
     default_visibility = ["//visibility:public"],
     licenses = ["notice"],  # Apache 2.0
 )
 
+td_library(
+    name = "lce_ops_td_file",
+    srcs = [
+        "ir/lce_ops.td",
+    ],
+    includes = ["include"],
+    deps = [],
```

Comment:
I think this should include the following dependencies to be correct:
```
"@org_tensorflow//tensorflow/compiler/mlir/lite/quantization:quantization_td_files",
"@llvm-project//mlir:SideEffectTdFiles",
```

Reply:
Done